packages/anthropic/src/__snapshots__/anthropic-messages-language-model.test.ts.snap

```diff
@@ -15,6 +15,7 @@ exports[`AnthropicMessagesLanguageModel > doGenerate > agent skills > should exp
         },
       ],
     },
+    "contextManagement": null,
```

Comment:
please add a couple of test cases under generate where you unit test the context management mapping to a request body